src/DependencyInjection/TacticianDomainEventExtension.php

```diff
@@ -71,4 +77,17 @@ private function addSubscribers(ContainerBuilder $container)
             ]);
         }
     }
+
+    private function registerEventCollector(ContainerBuilder $container, $useIdentityMap)
+    {
+        $class = 'BornFree\TacticianDoctrineDomainEvent\EventListener\CollectsEventsFromEntities';
+        if ($useIdentityMap === true) {
```

Comment:
just `if ($useIdentityMap)` does the same I guess, we have a `false` default value. It can't be anything else ;)